### PR TITLE
linux-capability: add note for using CAP_DAC_READ_SEARCH in docker container

### PR DESCRIPTION
**Pull request by Watson1978**
Related to https://github.com/fluent/fluentd/issues/4213

### CODE REVIEW

**Review by daipom — COMMENTED** [review]
Thanks so much for this enhancement!
It looks basically good to me, but how about moving this section as follows?

I feel a separate section would be easier to understand because this is not directly related to `in_tail`.

Screenshot from 2024-12-23 18-08-58

File: deployment/linux-capability.md [hunk 1]
```diff
@@ -84,6 +84,21 @@ Inheritable: dac_override, dac_read_search
 Permitted:   dac_override, dac_read_search
 ```
 
+#### Note: using CAP\_DAC\_READ\_SEARCH in docker container
+
+When using `CAP_DAC_READ_SEARCH` in a Docker container, you need to add the `--cap-add DAC_READ_SEARCH` option to the `docker run` command.
+Or, if you are using `docker compose`, you need to add `cap_add` to the service definition.
```
Comment (kenhys) [inline deployment/linux-capability.md]: It might be better to explain "the context" - such as collecting logs with non-root user.

**Review by kenhys — CHANGES_REQUESTED** [review]

**Comment by Watson1978** [comment]
Thanks. 
I moved this to last section and add some explain.

**Review by kenhys — APPROVED** [review]
LGTM.

**Review by daipom — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**